app/Http/Controllers/Admin/SiteSettingsController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use Notifications;
+use App\Http\Requests\AdminSiteSettingsRequest;
+
+class SiteSettingsController extends Controller
+{
+    public function __construct()
+    {
+        \Title::prepend(trans('dashboard.title.prepend'));
+    }
+
+    public function getSettings()
+    {
+        \Title::append(trans('settings.index.title'));
+
+        return view('admin.settings.index');
+    }
+
+    public function setSettings(AdminSiteSettingsRequest $request)
```

Comment:
let's call it `saveSettings`.
`set` and `get` are very specific words for setters and getters